djangosaml2/views.py

```diff
@@ -89,6 +90,18 @@ def _get_subject_id(session):
         return None
 
 
+def _get_next_path(request: HttpRequest) -> Optional[str]:
+    if "next" in request.GET:
+        next_path = request.GET["next"]
+    elif "RelayState" in request.GET:
+        next_path = request.GET["RelayState"]
```

Comment:
what about having an base64url encoded relaystate?

Reply:
It just raised in my mind but if you don't see this as interesting, since it was there before, don't mind

Reply:
I'd like to imitate the functionality of the login endpoint as closely as possible.